src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ListItem.css';
+import '../../components/Item/Item';
+import Item from '../../components/Item/Item';
+function ListItem() {
+  return (
+    <div id = "ListCenter">   
+      <ul className="container flex">
+        <li data-testid = "TestItem" className="item flex-item"><Item /></li>
```

Comment:
Put Items in an array and render each array item. Each item should have an unique key. The array index can be used as a key.